devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusGenerateCode.java

```diff
@@ -29,6 +29,7 @@
 
     public static final String QUARKUS_GENERATED_SOURCES = "quarkus-generated-sources";
     public static final String QUARKUS_TEST_GENERATED_SOURCES = "quarkus-test-generated-sources";
+    public static final String[] CODE_GENERATION_PROVIDER = new String[] { "grpc" };
```

Comment:
could you add a `TODO` to get this list dynamically (if it's not easy enough to just do it ;) )?